vllm/core/block_manager.py

```diff
@@ -269,6 +269,10 @@ def free(self, seq: Sequence) -> None:
         self.block_tables[seq_id].free()
         del self.block_tables[seq_id]
 
+    def free_seq_cached_tokens(self, seq: Sequence) -> None:
```

Comment:
```suggestion
    def remove_seq_from_computed_blocks_tracker(self, seq: Sequence) -> None:
```
free_seq_cached_tokens is a little ambiguous as people may think this function is freeing the block of some cached tokens in block_manager.
Does this function name look better for you?